etc/helm/pachyderm/templates/_helpers.tpl

```diff
@@ -89,7 +89,7 @@ http://localhost:4000/oauth/callback/?inline=true
 
 {{- define "pachyderm.pachdRedirectURI" -}}
 {{- if .Values.pachd.oauthRedirectURI -}}
-{{ .Values.console.config.oauthRedirectURI -}}
```

Comment:
Good catch! This looks like an unrelated bug?

Reply:
Yeah I could break it out into its own PR too

Reply:
I think that'd be good from a release notes perspective